addons/html_builder/static/src/core/building_blocks/builder_row.scss

```diff
@@ -1,15 +1,18 @@
 @mixin sublevel-line($_level-left: 0) {
     position: absolute;
+    top: 0;
+    bottom: 0;
+    left: $_level-left - $o-we-border-width;
     border: $o-we-border-width solid var(--o-hb-row-sublevel-color, #{mix($o-we-bg-lighter, $o-we-fg-light)});
     border-width: 0 0 $o-we-border-width $o-we-border-width;
-    height: calc(100% + #{$o-hb-row-spacing * 3});
     pointer-events: none;
-    transform: translate($_level-left - $o-we-border-width, ($o-hb-row-min-height * -0.65) - $o-we-border-width);
+    transform: translate(0, (($o-hb-row-min-height + $o-hb-row-spacing) * -0.5) + $o-we-border-width);
     content: "";
 }
 
 .hb-row {
     $_z-index: 4;
+    position: relative;
```

Comment:
Is it really necessary, it seems `hb-row` elements already have the `position-relative`class, could you double check ?

<img width="507" height="455" alt="Capture d’écran 2025-09-30 à 13 29 01" src="https://github.com/user-attachments/assets/665f23ec-94fb-4764-a42e-63650035546b" />